sources/packages/web/src/components/generic/Banner.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <v-banner lines="two" :class="bannerClass">
```

Comment:
I know that we talked about v-banner in the meeting because if was the first thing that came to our mind during the meeting, but vuetify has the below component that looks more with what we need. Did you check it?
https://vuetifyjs.com/en/components/alerts/#outlined
![image](https://user-images.githubusercontent.com/61259237/168878228-cc89d9b4-59de-4c0f-9315-2f05f636378e.png)

Reply:
I did not know about it, I will look into it.